cmd/bootkube/render.go

```diff
@@ -43,6 +44,7 @@ func init() {
 	cmdRender.Flags().StringVar(&renderOpts.caCertificatePath, "ca-certificate-path", "", "Path to an existing PEM encoded CA. If provided, TLS assets will be generated using this certificate authority.")
 	cmdRender.Flags().StringVar(&renderOpts.caPrivateKeyPath, "ca-private-key-path", "", "Path to an existing Certificate Authority RSA private key. Required if --ca-certificate is set.")
 	cmdRender.Flags().StringVar(&renderOpts.etcdServers, "etcd-servers", "http://127.0.0.1:2379", "List of etcd servers URLs including host:port, comma separated")
+	cmdRender.Flags().StringVar(&renderOpts.etcdStorageType, "etcd-storage-type", "", "")
```

Comment:
I actually intend to let @aaronlevy to decide naming.
This is just to show the workflow.